include/cadmium/engine/pdevs_dynamic_coordinator.hpp

```diff
@@ -94,56 +93,72 @@ namespace cadmium {
                     }
 
                     // Generates structures for direct access to external couplings to not iterate all coordinators each time.
+                    bool found;
```

Comment:
I don't see why you define it out of the for scope. Also, the code below looks like very good place to use std::find family of functions

Reply:
I define the found variable out the for scope because i continue using it after the for, but as you suggested I will replace this code to use std::find instead in order to be more legible. Added to my todo list.